repos/spack_repo/builtin/packages/py_darshan/package.py

```diff
@@ -56,6 +57,8 @@ class PyDarshan(PythonPackage):
     for v in ["3.4.0", "3.4.1", "3.4.2", "3.4.3", "3.4.4", "3.4.5", "3.4.6", "3.4.7"]:
         depends_on(f"darshan-util@{v}", when=f"@{v}", type=("build", "run"))
 
+    depends_on("darshan-util@3.5.0", when="@3.5.0", type=("build", "run"))
```

Comment:
```suggestion
for v in ["3.4.0", "3.4.1", "3.4.2", "3.4.3", "3.4.4", "3.4.5", "3.4.6", "3.4.7", "3.5.0]:
depends_on(f"darshan-util@{v}", when=f"@{v}", type=("build", "run"))
```

We can just add the following instead of creating a full new line.

Reply:
I just thought line 57 will grow quickly.
In the future, such dependency will be added for each new release.
Is there a better way in python for such task?